draft-selander-ace-ake-authz.md

```diff
@@ -369,6 +369,113 @@ TODO: The voucher response may contain a "Voucher-info" field as an alternative
 
 This section defines the profile of the ACE framework (see Appendix C of {{I-D.ietf-ace-oauth-authz}}).
 
+U plays the role of the ACE Resource Server (RS).
+V plays the role of the ACE Client (C).
+W plays the role of the ACE Authorization Server (AS).
+
+C and RS use the Auxiliary Data in the LAKE protocol to communicate.
+C and RS use the LAKE protocol to protect their communication.
+LAKE also provides mutual authentication of C and RS, assisted by the AS.
+
+## Protocol Overview
+
+~~~~~~~~~~~
+   RS                                C                     AS
+   |          LAKE Message 1         |                     |
+   |  AD1=AS Request Creation Hints  |                     |
+   |-------------------------------->|     POST /token     |
+   |                                 |-------------------->|
+   |                                 |                     |
+   |                                 | Access Token +      |
+   |           LAKE Message 2        |  Access Information |
+   |          AD2=Access Token       |<--------------------|
+   |<--------------------------------|                     |
+   |           LAKE Message 3        |                     |
+   |-------------------------------->|                     |
+
+~~~~~~~~~~~
+{: #fig-mapping-ace title="Overview of the protocol mapping to ACE" artwork-align="center"}
+
+RS proactively sends the AS Request Creation Hints message to C to signal the information on where C can reach the AS.
+RS piggybacks the AS Request Creation Hints message using Auxiliary Data of the LAKE message 1.
+Before continuing the LAKE handshake, based on the AS Request Creation Hints information, C sends a POST request to the token endpoint at the AS requesting the access token.
+The AS issues to C a Bearer token that is cryptographically protected based on the secret shared between the AS and RS.
```

Comment:
I would rather we wrote: "AS issues an object to C that is ...  It is encoded using the Bearer Token container."